src/libcore/task/wake.rs

```diff
@@ -131,9 +129,14 @@ impl RawWakerVTable {
     /// resources that are associated with this instance of a [`RawWaker`] and
     /// associated task.
     #[rustc_promotable]
-    #[unstable(feature = "futures_api",
-            reason = "futures in libcore are unstable",
-            issue = "50547")]
+    #[cfg_attr(stage0, unstable(feature = "futures_api_const_fn_ptr", issue = "50547"))]
+    #[cfg_attr(not(stage0), stable(feature = "futures_api", since = "1.35.0"))]
+    // `rustc_allow_const_fn_ptr` is a hack that should not be used anywhere else
+    // without first consulting with T-Lang.
+    //
+    // FIXME: remove whenever we have a stable way to accept fn pointers from
+    // const fn (see rfcs/pull/2632)
```

Comment:
Doesn't that RFC only deal with trait methods, and at most "const fn pointers", as in, pointers to `const fn`s?
Whereas here, the pointers would be to runtime functions, not `const fn`s.

Also, please add the org name (`rust-lang/rfcs/pull/2632`), or use the hash notation (rust-lang/rfcs#2632), or even include the whole link.

Reply:
Well it's future work atm https://github.com/oli-obk/rfcs/blob/const_generic_const_fn_bounds/text/0000-const-generic-const-fn-bounds.md#future-work but it was relevant to cc so that we may want to expedite function pointers and consider them at the same time or in a sibling RFC.

Reply:
It's "`const fn` pointers", not `fn` pointers passed to `const fn`, AFAICT: https://github.com/oli-obk/rfcs/blob/const_generic_const_fn_bounds/text/0000-const-generic-const-fn-bounds.md#const-function-pointers

Reply:
Sure; we'd need a `x: ?const fn()` syntax for the latter much like `T: ?const MyTrait`.

Reply:
@cramertj this was not resolved, the RFC is not really relevant to the problem at hand (promotable `const fn`s aren't happening *anyway*, you really need pattern aliases or similar, see all my other comments).

Reply:
I've seen your comments-- I'm not sure I understand. Is there an objection you're raising here that wasn't covered in the discussion [on the Future PR?](https://github.com/rust-lang/rust/pull/59119#discussion_r264930443) We had discussed the issues with this approach, including that the long-term story for const-fn-promotion is unclear, but we already expose it in existing standard library functions and it's necessary for making this API work well.

Reply:
I think the complaint is that the fixme points to the bounds on generic const fn RFC, but that RFC doesn't introduce those (it just mentions them). Basically just change the link to point to https://github.com/rust-rfcs/const-eval/issues/19#issuecomment-472799062

Reply:
Done!